hugolib/page_permalink_test.go

```diff
@@ -59,6 +59,8 @@ func TestPermalink(t *testing.T) {
 
 		// test URL overrides
 		{"x/y/z/boofar.md", "", "", "/z/y/q/", false, false, "/z/y/q/", "/z/y/q/"},
+		// test URL override with expands
+		{"x/y/z/boofar.md", "", "test", "/z/:slug/", false, false, "/z/test/", "/z/test/"},
```

Comment:
It's a little hard to see that this new feature works without a integration test.

I suggest that you create one in `page/permalinks_integration.test` -- look for examples in test files with similar names. They use the `hugolib.Test` function.

Reply:
I did leave this test and implemented a new one in permalinks_integration_test.